### PR TITLE
Made web_widget_color installable and remove the search field.

### PR DESCRIPTION
**Pull request by llacroix**
The search field doesn't seem to exist anymore. Without it, the module loads correctly. And it seems to work fine.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you take the occasion of including some demo data (inherit an existing view to put this widget in a field susceptible to use it)? This way, it can be easy to try.

**Comment by llacroix** [comment]
Sure

File: web_widget_color/__openerp__.py
Comment (elicoidal) [inline web_widget_color/__openerp__.py:22]: Hi @llacroix  thanks for your contribution
I will check mostly format and guidelines in your PR.
- Please use simple headers following OCA templates
  https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3
- Adapt the version number: 9.0.1.0.0

Comment (elicoidal) [inline web_widget_color/__openerp__.py]: Can you include also a README file, following: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

**Comment by njeudy** [comment]
@llacroix think it's not sufficient to work .. can you give me acces to your branch to help you for this ?

**Comment by njeudy** [comment]
@llacroix I correct this module to work in tree view too . can you give me acces to your branch ? or should I fork your branch ?

Thanks.

**Comment by njeudy** [comment]
I update tree view widget with a small square with color.

Square color on tree_view

Code is ready just says to me where I can push :)

**Comment by llacroix** [comment]
That's cool, I'll check for access

**Comment by llacroix** [comment]
@njeudy  You should be able to push on my fork in this branch.

**Comment by njeudy** [comment]
@llacroix ok commit pushed :) now work on form view and tree view (editable bottom)

tested on community and entreprise.

**Comment by gurneyalex** [comment]
:+1:

**Comment by lmignon** [comment]
:+1:

**Comment by pedrobaeza** [comment]
There are some things unattended:
- Bump version number to 9.0.1.0.0
- Short headers
- Demo data

So please don't merge.

**Comment by lmignon** [comment]
@pedrobaeza 

> Bump version number to 9.0.1.0.0

For sure

> Short headers
> Demo Data

IMO both are nice to have. We can't require from the guys who spend time to migrate a module to fix this kind of cosmetic change. The same applies to the demo data. For sure it would be better to have also these changes but these are not blockers.

**Comment by elicoidal** [comment]
@lmignon I understand this is extra workload but we need to do it at some point and if not now, it will never be done.
So I would be in favor

**Comment by pedrobaeza** [comment]
OK about demo data. It's optional but very needed in this case, because this widget can't be tried directly in runbot if we don't provide demo data. About short headers, I consider also a must have for 9.0.

**Comment by sbidoul** [comment]
Tested. Works fine on v9 community.

@llacroix @njeudy would you have a moment to fix the version (9.0.1.0.0) and the headers (as well as the list of contributors in README.rst). 

The demo data is nice to have but not blocking for me.

**Comment by lepistone** [comment]
Thanks for the work! Please let me know if I can help with what was requested (headers + version number and some demo data).

It would be nice to merge this since mis_builder OCA/account-financial-reporting#189 depends on it.

**Review by yelizariev — APPROVED** [review]
I have checked it on runbot by adding `widget="color"` to field street2 of partner form.

http://3154933-270-9ae936.runbot2.odoo-community.org/web?#id=6&view_type=form&model=res.partner

Also, requested style updates are done.

:+1:

File: web_widget_color/__openerp__.py
Comment (pedrobaeza) [inline web_widget_color/__openerp__.py]: Change to `coding`

File: web_widget_color/__init__.py
Comment (pedrobaeza) [inline web_widget_color/__init__.py]: Change to `coding`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by lasley — APPROVED** [review]
Other than the @pedrobaeza comments - 👍  code, no test

**Comment by yelizariev** [comment]
@pedrobaeza requested changes are done

**Comment by moylop260** [comment]
Please review lints output
https://travis-ci.org/OCA/web/jobs/163172631#L442

**Comment by sbidoul** [comment]
Last fixes are in https://github.com/llacroix/web/pull/3 @llacroix can you merge into your branch?

**Comment by pedrobaeza** [comment]
Closing as superseeded.

**Pull request closed without merging**
